### PR TITLE
Notification for update listing and QnA

### PR DESCRIPTION
**Pull request by Mik3Qiao**
- Added notification when update a listing
- Added notification when creating a new question for a listing
- Added notification when creating a new comment for a listing
- Display component is imported and made reusable now

### CODE REVIEW

**Comment by NathanCarriere** [comment]
Could screenshots be attached to this? Where would the notifications appear on the screen?

File: frontend/src/Util/DisplayWarning.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+export const DisplayInfo = (props) =>{
+  const {
+    displayMessage,
+    displaySuccess,
+    displayWarning,
+    successMessage,
+    failedMessage,
+    WarningMessage,
+    handleCloseMessage,
+  } = props
```
Comment (NathanCarriere) [inline frontend/src/Util/DisplayWarning.jsx:15]: Does this just add new items to props?

Comment (jcsnorlax97) [reply frontend/src/Util/DisplayWarning.jsx:15]: This is "destructuring" the props. If we don't use destructuring, how we access data inside props would be `props.failedMessage`, `props.WarningMessage`, `props.handleCloseMessage` and so on. With uses of destructuring, we can access them by simply calling `failedMessage`, `WarningMessage`, `handleCloseMessage` and so on. Another benefit of being this explicit is that this helps readers know what are in the props.

Comment (jcsnorlax97) [reply frontend/src/Util/DisplayWarning.jsx:15]: (Additional comment: this does not add new items to props. it only demonstrates what have been passed from caller, i.e. parent component, into this DisplayInfo component.

Comment (NathanCarriere) [reply frontend/src/Util/DisplayWarning.jsx:15]: Interesting.

**Review by NathanCarriere — APPROVED** [review]

**Review by RonnieDsouza — APPROVED** [review]
👍

**Pull request merged**